build/tools/PrepareRelease/publish-all.bat

```diff
@@ -1,13 +1,2 @@
-
-REM SET SOLUTION_DIR=C:\Github\dd-trace-dotnet
-
-SET TOOL_BUILD_CONFIG=Release
-SET INTEGRATIONS_PROJ=%SOLUTION_DIR%\src\Datadog.Trace.ClrProfiler.Managed\Datadog.Trace.ClrProfiler.Managed.csproj
-SET OUTPUT_DIR=%SOLUTION_DIR%\build\tools\PrepareRelease\bin\tracer-home
-
 RMDIR "%OUTPUT_DIR%" /S /Q
-
-dotnet publish %INTEGRATIONS_PROJ% -c %TOOL_BUILD_CONFIG% -f net45 -o "%OUTPUT_DIR%\net45"
```

Comment:
Note: removing these `dotnet publish` calls is what initially motivated this PR.